en/concepts/scene/skybox.md

```diff
@@ -0,0 +1,55 @@
+# Skybox
+
+__Skybox__, in video games, is a cube that wraps the entire scene. The __Skybox__ can render the atmosphere of the entire environment and express the environment of the entire scene. The __Skybox__ is also a very important part in the __PBR-based workflow__.
+
+The __Skybox__ in __Cocos Creator 3D__, is shown below:
+
+![skybox](skybox/Skybox.jpg)
+
+## Enabling Skybox
+
+Enabling the __Skybox__ effect in __Cocos Creator 3D__, takes just one step:
+
+  - The __Skybox__ panel is on the Scene node's __Properties panel__. Check the *Enabled* property to enable the __Skybox__.
+  ![Open skybox](skybox/SkyboxPanel.jpg)
+
+ > **__Note:__ When the *Envmap* property of __Skybox__ is empty, the pixel map will be used and displayed**.
+
+## Modify the environment map of the Skybox
+
+Modifying the environment map of the __Skybox__ in __Cocos Creator 3D__ is done by setting a resource of type __TextureCube__. Importing from a resource into a __TextureCube__ and setting it into a __Skybox__ can be done in the following steps:
+
+  1. Import picture resources. (**Note: Here is an example of a panorama, here is how to make a CubeMap**.)
```

Comment:
Here is an example of a panorama, here is how to make a CubeMap ->
Here is an example of a panorama, we will introduce how to make a CubeMap later in this documentation